app/_includes/md/enterprise/cassandra-deprecation.md

```diff
@@ -0,0 +1,6 @@
+{:.warning .no-icon}
+> **Deprecation warning:** Support for Cassandra is deprecated with 
```

Comment:
```suggestion
> **Deprecation warning:** Cassandra is deprecated with 
```

Reply:
Using "support for" can be misinterpreted by customers and is often a loaded term. Let's simply remove it to reduce confusion.

Reply:
done